packages/models-library/src/models_library/services.py

```diff
@@ -482,6 +483,11 @@ class ServiceDockerData(ServiceKeyVersion, _BaseServiceCommonDataModel):
         alias="boot-options",
         description="Service defined boot options. These get injected in the service as env variables.",
     )
+    min_visible_inputs: NonNegativeInt | None = Field(
+        None,
+        alias="min-visible-inputs",
```

Comment:
Q?  why do we need an alias here? why can't we simply call the entry `min_visible_inputs`? 

why to introduce `-` in the field names (I know it was already done in `boot-options` or `integration-version` but I also thing is inconvenient)? We already struggle with alias because of differences in camel-case vs snake case. In  addition, this alias cannot be used as variable name. 

Now we have the three options mixed depending on the entrypoint i.e.
- ``my_field``
- ``my-field``
- ``myField``

I think we should keep it as simple as possible and avoid adding unnecessary overhead that either complicates or uses extra computational energy (even if it is epsilon!) :-)

Reply:
some inputs on these? :-)